i18n/zh-hant.php

```diff
@@ -0,0 +1,347 @@
+          <br /><br />
+          <div class="col-md-7">
+            <div class="button" data-toggle="buttons">
+              <label class="btn btn-default btn-block" onclick="$('#iup').show();$('#iatt').show();"><input type="radio" id="s21" name="s2" value="1" />我或其他人已經將檔案上傳至維基共享資源</label>
+              <label class="btn btn-default btn-block" onclick="$('#iup').hide();$('#iatt').show();"><input type="radio" id="s22" name="s2" value="2" />我將會在電子郵件中附上檔案</label>
+            </div>
+            <br />
+            <p>如果您還沒有上傳檔案到維基共享資源，請使用<a href="//commons.wikimedia.org/wiki/Special:UploadWizard" target="_blank"><b>上傳精靈</b></a>進行上傳。</p>
+            <p>為了防止您的檔案在維基媒體 OTRS 團隊審核前遭到刪除，您可能需要在該檔案的描述頁面增加 <b><samp>{{subst:OP}}</samp></b> 文字。</p>
+            <br /><br />
+          </div>
+          <div class="col-md-4">
+            <div style="display:none;" id="iatt">
+              <div style="display:none;" id="iup">
+                維基共享資源的檔案名稱<br /><input type="text" name="filer" placeholder="範例.jpg (必填)" class="form-control" />
+                <br /><br />
+              </div>
+              <a role="button" href="#s3" class="btn btn-primary btn-block smsc nt">前進至下一步</a>
+            </div>
+          </div>
+          <br />
+        </div>
+
+        <div id="s3" class="row hof"> <!-- step 3 -->
+          <br />
+          步驟 3/5 [ <a href="#s2" class="smsc">返回</a> | <a href="//commons.wikimedia.org/wiki/Commons:Help_desk?action=edit&section=new&preloadtitle=help+with+Wikimedia+OTRS+release+generator+step+3" target="_blank">說明</a> | <a href="//commons.wikimedia.org/wiki/User_talk:FDMS4?action=edit&section=new&preloadtitle=Wikimedia+OTRS+release+generator+feedback" target="_blank">回饋</a> ]
+          <br /><br />
+          <div class="col-md-7">
+            <div class="button" data-toggle="buttons">
+              <label class="btn btn-default btn-block" onclick="$('#i3').show();"><input type="radio" id="s31" name="s3" value="1" />我要釋出媒體作品</label>
+              <label class="btn btn-default btn-block" onclick="$('#i3').show();"><input type="radio" id="s32" name="s3" value="2" />我要釋出從某個媒體作品當中所創作的作品</label>
+              <label class="btn btn-default btn-block" onclick="$('#i3').show();"><input type="radio" id="s33" name="s3" value="3" />我要釋出媒體作品與從其作品當中創作的作品</label>
+            </div>
+            <br />
+            <p>如果您的作品是從某人的作品以複雜方式進行創作或將某人的作品包含其中，那麼該作品就是一件<a href="//commons.wikimedia.org/wiki/Commons:Derivative_works" target="_blank"><b>衍生創作作品</b></a>，一般必須要提供該作品與其原始作品的著作權持有者釋出許可。</p>
+            <p>在部分國家，由於有<a href="//commons.wikimedia.org/wiki/Commons:Freedom_of_panorama" target="_blank"><b>全景自由攝影</b></a>的例外，建築物或其他永久公開展示的作品可不用提供原始著作權持有者釋出許可。在美國，只有建築物不需要提供原始著作權持有者的許可。</p>
```

Comment:
See also https://commons.wikimedia.org/wiki/Commons:Freedom_of_panorama#Taiwan_.28Republic_of_China.29. You may want to add a note on this.